docs/api-conventions.asciidoc

```diff
@@ -233,7 +233,7 @@ Client methods can throw two kinds of exceptions:
 
 * Requests that were received by the {es} server but that were rejected 
 (validation error, server internal timeout exceeded, etc) will produce an 
-`ElasticsearchException`. This exception contains details about the error,
+`OpensearchException`. This exception contains details about the error,
```

Comment:
We don’t have to update anything inside docs. Since we will be removing this folder completely